src/config.ts

```diff
@@ -79,12 +80,15 @@ export enum RuntimeTypes {
 
 const emeraldConfig: ParaTimeConfig = {
   mainnet: {
+    address: 'oasis1qzvlg0grjxwgjj58tx2xvmv26era6t2csqn22pte',
```

Comment:
what was the discussion on hard coding these? gonna scroll around in this PR to see if there's any

Reply:
There was a comment that these values can be hardcoded.
https://github.com/oasisprotocol/oasis-wallet-web/pull/969#discussion_r944747406

related commit: https://github.com/oasisprotocol/oasis-wallet-web/pull/969/commits/12034bdad2067f81e689896efa740d9319788678

Reply:
merely that we can. it's not like they'll change, after all. and it saves the extra awaiting and surely there's some CPU time saved. and it's definitely helpful for that to be grep-able

maybe add a comment with the code for doing the derivation so that we have it handy when adding new paratimes

Reply:
I like that it's searchable; and I don't know how the derivation would best be described and named. But hardcoding does add a possibility of mistakes :/ Add a test?
```ts
describe('paraTimesConfig', () => {
  it('addresses should be derived from runtimeId', async () => {
    for (const paraTimeConfig of Object.values(paraTimesConfig)) {
      for (const paraTimeNetwork of [paraTimeConfig.mainnet, paraTimeConfig.testnet, paraTimeConfig.local]) {
        if (paraTimeNetwork.runtimeId) {
          const address = await staking.addressFromRuntimeID(misc.fromHex(paraTimeNetwork.runtimeId))
          expect(staking.addressToBech32(address).toLowerCase()).toEqual(paraTimeNetwork.address)
        }
      }
    }
  })
})
```